great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -161,7 +163,8 @@ def __init__(self, table_name=None, engine=None, connection_string=None):
 
     def add_default_expectations(self):
         """
-        The default behavior for PandasDataset is to explicitly include expectations that every column present upon initialization exists.
+        The default behavior for SqlAlchemyDataset is to explicitly include expectations that every column present upon
```

Comment:
Should we move this to base.py?

Reply:
I actually just had to move it out of there...the behavior for getting columns is slightly different in pandas and sqlalchemy. The previous implementation was in base, but was dependent on the pandas semantics.